test/framework/config_transparentproxy.go

```diff
@@ -80,7 +80,7 @@ var defaultTProxyConf = TransparentProxyConfig{
 	IPV6: false,
 }
 
-func init() {
+func InitTproxyConfig() {
```

Comment:
If it's used in a single place what's the point of abstracting this in the framework far from where it's actually used?

Reply:
You are right, I'm gonna followup with the PR removing it from the framework (it's blocking other PRs, so I'm not gonna include this in this PR to merge it faster)